Marlin/src/HAL/STM32F1/inc/SanityCheck.h

```diff
@@ -39,7 +39,7 @@
   #error "SERIAL_STATS_DROPPED_RX is not supported on the STM32F1 platform."
 #endif
 
-#if ENABLED(NEOPIXEL_LED)
+#if ENABLED(NEOPIXEL_LED) && DISABLED(MKS_MINI_12864_V3)
   #error "NEOPIXEL_LED (Adafruit NeoPixel) is not supported for HAL/STM32F1. Comment out this line to proceed at your own risk!"
```

Comment:
If Neopixel is now working with HAL/STM32F1, that is news to me.

Reply:
Sorry, maybe I told you earlier, my latest test situation.
1. In pins_MKS_ROBIN_E3_common.h fix pin
```
#elif ENABLED(MKS_MINI_12864_V3)    
    #define DOGLCD_CS                       PA4
    #define DOGLCD_A0                       PA5
    #define LCD_PINS_DC                     DOGLCD_A0
    #define LCD_BACKLIGHT_PIN               -1
    #define LCD_RESET_PIN                   PA6
    #define NEOPIXEL_PIN                    PA7
    #define DOGLCD_MOSI                     PB15
    #define DOGLCD_SCK                      PB13
    #define FORCE_SOFT_SPI
```

2. I has test it and display is ok on E3 board， but there is a bug .
A. If i have enable #define FORCE_SOFT_SPI, the lcd blacklight's color is can be change by lcd, but read card by lcd and it will display stuck 
B. If i have disenable #define FORCE_SOFT_SPI, the lcd blacklight's color is can not be change by lcd, but read card by lcd is OK